{
  Assert.fail("Error in client while talking to server");
}
